src/woo_publications/publications/migrations/0007_publication_opsteller_publication_publisher_and_more.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 4.2.16 on 2024-11-06 15:24
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("metadata", "0007_alter_organisation_is_actief"),
+        ("publications", "0006_merge_20241101_1532"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="publication",
+            name="opsteller",
+            field=models.ForeignKey(
+                blank=True,
+                help_text="The organisation which drafted the publication and it's content.",
+                limit_choices_to={"is_actief": True},
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="publication_drafters",
+                to="metadata.organisation",
+                verbose_name="drafter",
+            ),
+        ),
+        migrations.AddField(
+            model_name="publication",
+            name="publisher",
+            field=models.ForeignKey(
+                default="",
```

Comment:
I think this may cause crashes on existing data? Since a string is not a valid type for an integer-based foreign key. You should easily be able to test this in your dev environment by creating a publication on the main branch and then checking this out and migrating.

If it causes a crash, you can add a `RunPython` operation first to create an organisation with ID `1` if it doesn't exist yet. The data isn't semantically correct then, but as you said, we're not on 1.0 or in production anywhere yet so it's fine :)

```suggestion
                default=1,
```